pkgs/applications/networking/powerdns-admin/default.nix

```diff
@@ -95,6 +101,16 @@ in stdenv.mkDerivation {
     substituteInPlace powerdnsadmin/__init__.py --replace "sess = Session(app)" "app.config['SESSION_SQLALCHEMY'] = models.base.db; sess = Session(app)"
     # Routes creates session database tables, so it needs a context
     substituteInPlace powerdnsadmin/__init__.py --replace "routes.init_app(app)" "with app.app_context(): routes.init_app(app)"
+
+    # flask 2.3: 'Blueprint' object has no attribute 'before_app_first_request'
+    # Run code before app.run() instead
+    substituteInPlace run.py \
+      --replace 'from powerdnsadmin import create_app' 'from powerdnsadmin import create_app
+    from .powerdnsadmin.routes.index import register_modules' \
+      --replace 'app = create_app()' 'app = create_app()
+        with app.app_context():
+            register_modules()'
+    ${all_post_patch}
```

Comment:
We are accumulating a rather large amount of substituteInPlace here which change code. We should really convert them to patches to notice breakages and changes in the code immediately.